### ISSUE TITLE
CTC beam search with charlm

### ISSUE THREAD

**Issue by cyu0913** [hirofumi0810/tensorflow_end2end_speech_recognition#30]
Hi,

This is very interesting project.

I have a question on the CTC beam search (with charlm) you're currently implementing. Are you planning to use RNN based character language model or n-gram based one? I think it will be very useful to have CTC beam search that support RNN based character language model.

Thanks.

**Comment by hirofumi0810** [comment]
Yes, I'm planning to add CharRNNLM, but I'm working on Pytorch framework now.
So, please give me a time (1-2 months).

**Comment by cyu0913** [comment]
That's awesome! Thank you!

**Comment by GabrielLin** [comment]
What about the progress? I think it is a great function. Thanks.
